test/receiveAndDeleteMode.spec.ts

```diff
@@ -157,7 +159,9 @@ describe("Streaming Receiver from Queue/Subscription", function(): void {
         return Promise.resolve();
       },
       (err: Error) => {
-        should.not.exist(err);
+        if (err) {
```

Comment:
Do we need the if check? The callback would executes only if there was an error.

Can replace with:

uncaughtErrorFromHandlers = err;

Reply:
Updated this check inside the `assert.fail`, so we dont need this anymore.
We had this to get around the linter error which tells that err can possibly be undefined